### PR TITLE
Unit testing math challenge

### PR DESCRIPTION
**Pull request by itsTHL**

### CODE REVIEW

File: js/js-unit-testing/js-unit-testing_math-functions/index.test.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import { add } from "./index.js";
```
Comment (jamoragar) [inline js/js-unit-testing/js-unit-testing_math-functions/index.test.js:1]: you can import all the methods in one line, separated by coma 😁

**Review by jamoragar — APPROVED** [review]
Well done!

File: js/js-unit-testing/js-unit-testing_math-functions/index.test.js (hunk 1 above)
Comment (jamoragar) [reply js/js-unit-testing/js-unit-testing_math-functions/index.test.js:1]: `import { add, subtract, multiply, divide } from "./index.js";`

Comment (itsTHL) [reply js/js-unit-testing/js-unit-testing_math-functions/index.test.js:1]: Thanks!

**Pull request merged**